insight/src/main/java/info/nightscout/androidaps/plugins/pump/insight/LocalInsightPlugin.java

```diff
@@ -836,7 +829,7 @@ public PumpEnactResult cancelTempBasal(boolean enforceNew) {
         PumpEnactResult cancelEBResult = null;
         if (isFakingTempsByExtendedBoluses()) cancelEBResult = cancelExtendedBolusOnly();
         PumpEnactResult cancelTBRResult = cancelTempBasalOnly();
-        result.success((cancelEBResult != null && cancelEBResult.getSuccess()) && cancelTBRResult.getSuccess());
+        result.success((cancelEBResult == null || (cancelEBResult != null && cancelEBResult.getSuccess())) && cancelTBRResult.getSuccess()); //Fix a bug when Fake TBR is disabled and click on Cancel TBR button
```

Comment:
@TebbeUbben Here I propose this modification to fix a bug in current driver:
=> If Fake TBR is disable in setting, each time you click on "Cancel TBR" you get a communication Error Popup